ui/src/app/pages/workspace/workspace-wrapper/component.ts

```diff
@@ -169,9 +168,6 @@ export class WorkspaceWrapperComponent implements OnInit, OnDestroy {
         this.accessLevel = workspace.accessLevel;
       }
     }));
-    this.subscriptions.push(profileStore.subscribe(({profile}) => {
```

Comment:
Wow, was this dropdown really the only thing that used this?

Reply:
yeah.. turns out a lot of the stuff in here is for one offs like this. will be gone soon!